flow/tutorials/TutorialScripts/drt/drc_fix.tcl

```diff
@@ -11,25 +11,25 @@ set_routing_layers -signal met1-met5
 set_macro_extension 2
 set route_guide [make_result_file route.guide]
 global_route -guide_file $route_guide \
-               -congestion_iterations 100 \
-               -verbose
+             -congestion_iterations 100 \
+             -verbose
 
 source ../../../platforms/sky130hd/setRC.tcl
 set_propagated_clock [all_clocks]
 estimate_parasitics -global_routing
 
 #detail_route
-set_thread_count 2
+set_thread_count 32
```

Comment:
Hardcoding 32 is definitely not the right answer here. A vast majority of users don't have that many threads.

Reply:
@rovinski Yes updated as follows
```
#2 CORES enough to run this design, change if needed
set_thread_count 2
```